api/sql/schema/750-rule.decision.fetch.sql

```diff
@@ -34,7 +34,7 @@ BEGIN
         countMonthly BIGINT
     )
 
-    SET @operationDate = ISNULL(@operationDate, GETDATE())
+    SET @operationDate = ISNULL(@operationDate, GETUTCDATE())
```

Comment:
This looks like a breaking change. If `@operationDate` parameter is in UTC, then `operationStartDate` and `operationEndDate` columns have to be in UTC too, which I am not sure is the case. Did you verify that?

Reply:
Probably could break current implementations. However, I think for sure we need all dates that are used for visualizing them in frontend to be UTC in database. We had huge amount of issues in previous old projects like ABZ where mobile apps cannot format to local datetime and we had to hardcode the timezone specifically for Zambia. Maybe we can do a major version in order to unblock ourselves as mobile developers are waiting the backend. @kalinkrustev , what is your opinion?